pallets/micropayment/src/tests.rs

```diff
@@ -322,6 +323,10 @@ fn signature() {
 
     let pk = Public::from_raw(pk);
     let sig = Signature::from_slice(&sig);
-    let verified = sr25519_verify(&sig, &msg, &pk);
+    let verified = sr25519_verify(
+        &sig.unwrap_or(UncheckedFrom::unchecked_from([0; 64])),
```

Comment:
Is it possible that someone can construct a `pub_key` and `msg` so the signature `[0; 64]` can always be verifyed? If yes I would suggest you return error when `from_slice` return none, if not then never mind.

Reply:
" [0u8; 64];" default signature value  in pallets/micropayment/src/lib.rs "pub fn verify_signature",  it is pass the production check.
```
            let mut sig = [0u8; 64];
            sig.copy_from_slice(&signature);
            let sig = sr25519::Signature::from_slice(&sig);
```

Reply:
If it worked in the past doesn't mean it will work in the future.

Reply:
Ok, let's keep it unchanged